### PR TITLE
AppVeyor: corrections for obsolete compilation messages

### PR DESCRIPTION
**Pull request by ladida28**
Hi Team,

I have checked the compilation messages in AppVeyor.
I adjusted WorkbookProtection.cs, XLCellTests.cs and XLWorkbookTests.cs to get rid of the obsolete messages.

Some changes were created by codemaid.

### CODE REVIEW

File: ClosedXML.Tests/Excel/Misc/XLWorkbookTests.cs [hunk 1]
```diff
@@ -359,7 +359,7 @@ public void WbProtect2()
             using (var wb = new XLWorkbook())
             {
                 var ws = wb.Worksheets.Add("Sheet1");
-                wb.Protect(true, false);
+                wb.Protect(password: String.Empty);
```
Comment (igitur) [inline ClosedXML.Tests/Excel/Misc/XLWorkbookTests.cs:362]: I think this test was indeed to test the obsolete method. We stil have to test those. It looks like you're duplicating the `WbProtect3` method here.

File: ClosedXML.Tests/Excel/Misc/XLWorkbookTests.cs [hunk 2]
```diff
@@ -406,7 +406,7 @@ public void WbProtect5()
             using (var wb = new XLWorkbook())
             {
                 var ws = wb.Worksheets.Add("Sheet1");
-                wb.Protect(true, false, "Abc@123");
+                wb.Protect("Abc@123");
```
Comment (igitur) [inline ClosedXML.Tests/Excel/Misc/XLWorkbookTests.cs:409]: I think this test was indeed to test the obsolete method. We stil have to test those. It looks like you're duplicating the `WbProtect3` method here.

Comment (ladida28) [reply ClosedXML.Tests/Excel/Misc/XLWorkbookTests.cs:409]: So what do you suggest, new pull request with the changes to WorkbookProtection.cs, XLCellTests.cs only?

File: ClosedXML.Tests/Excel/Misc/XLWorkbookTests.cs (hunk 1 above)
Comment (ladida28) [reply ClosedXML.Tests/Excel/Misc/XLWorkbookTests.cs]: changes reverted

**Comment by jahav** [comment]
Closing, because the proposed changes have already been applied in different commits (ad369b73fa7bae8b499c5dad7357868d76eb0b50, 1f58d1afc8dd0b03efbf92bbe4329abdb6ca9ed5).

**Pull request closed without merging**